### DISCUSSION TITLE
How to delete default template?

### DISCUSSION THREAD

**Discussion by cvitaa11** [okd-project/okd#886] [Q&A]
I want to show only my custom templates in the catalog, so I wanted to delete some default templates.
I see them all when I list them with `oc get templates -n openshift `, and then delete one with `oc delete template redis-ephemeral -n openshift` and I get success response but template is still present when I list them. Does anyone know how to solve this?

**Comment by vrutkovs** [comment]
Templates are provided by [cluster-samples-operator](https://github.com/openshift/cluster-samples-operator), these cannot be modified or removed
